arnold/models/database/case/workflow/balsamic.py

```diff
@@ -0,0 +1,31 @@
+"""Module for the workflow models."""
+from pydantic import BaseModel
+
+from arnold.models.database.case.mutliqc import (
+    Somalier,
+    PicardAlignmentSummary,
+    PicardDuplicates,
+    PicardHsMetrics,
+    PicardInsertSize,
+    PicardWGSMetrics,
+    SamtoolsStats,
+    Fastp,
+)
+from arnold.models.database.case.workflow.workflowinfo import WorkflowInfo
+
+
+class BalsamicSample(BaseModel):
+    sample_id: str
+    alignment_summary_metrics: PicardAlignmentSummary | None
+    duplicates: PicardDuplicates | None
```

Comment:
These fields are required but can be None. In what context would this be an expected behaviour? Shouldn't all of them be required except for wgs & somalier?